appcues/src/main/java/com/appcues/ui/primitive/ExperiencePrimitive.kt

```diff
@@ -60,7 +61,7 @@ internal fun ExperiencePrimitive.Compose() {
                 is ButtonPrimitive -> Compose(modifier)
                 is EmbedHtmlPrimitive -> Compose(modifier)
                 is HorizontalStackPrimitive -> Compose(modifier)
-                is ImagePrimitive -> Compose(modifier)
+                is ImagePrimitive -> Compose(modifier, matchParentBox)
```

Comment:
since we did this with Image in mind I think its ok to apply this only to image, but maybe we would want to replicate this to other types as well